docs/2.9.0/docs/canton/usermanual/kms/external_key_storage/participant_kms_migration/participant_kms_migration_new_namespace.rst

```diff
@@ -0,0 +1,54 @@
+..
+   Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates.
+..
+   Proprietary code. All rights reserved.
+
+.. _participant_kms_migration_new_namespace:
+
+Participant Node Migration with a New Namespace
+-----------------------------------------------
+
+To migrate a non-KMS participant to a KMS-enabled one we need to make sure we have this new participant
+connected to a KMS-compatible sync domain (e.g. :ref:`running JCE with KMS-supported encryption and
+signing keys <external_key_storage_interoperability>`).
+Afterwards, we transfer all parties, contracts and DARs from the old participant to the new one. Since, in this case,
+we do not transfer the identities from the old node to the new one we need to re-write our contracts
+so that they refer to the correct party ids. With this method, we can easily migrate our old participant
+that runs an older protocol version to a new participant with KMS enabled that can be running a more recent
+protocol version. Furthermore, we do not need to safe-keep the old node's root namespace key because
+our participant namespace changes.
+
+.. warning::
+    Currently this migration only works if you are migrating from an old participant node running protocol
```

Comment:
important also that this only works for a single operator or if all other participant operators agree and follow the same steps (which you did not test here).